### ISSUE TITLE
Mock Stats sample from POD is broken

### ISSUE THREAD

**Issue by atoomic** [CpanelInc/Test-MockFile#42]
when testing the code from the documentation:
https://metacpan.org/pod/release/TODDR/Test-MockFile-0.018/lib/Test/MockFile.pm#Mock-Stats

I can notice the following failure
```
Unknown args (3) passed to new at /usr/local/cpanel/3rdparty/perl/528/lib/perl5/cpanel_lib/Test/MockFile.pm line 385.
```

```perl
use  Test::MockFile;

my $mode = 0644;
my $now = time;
my $file = q[/tmp/foo];
my $contents = q[XYZ];

 Test::MockFile->new( $file, $contents, {
        'dev'       => 0,        # stat[0]
        'inode'     => 0,        # stat[1]
        'mode'      => $mode,    # stat[2]
        'nlink'     => 0,        # stat[3]
        'uid'       => 0,        # stat[4]
        'gid'       => 0,        # stat[5]
        'rdev'      => 0,        # stat[6]
        'atime'     => $now,     # stat[8]
        'mtime'     => $now,     # stat[9]
        'ctime'     => $now,     # stat[10]
        'blksize'   => 4096,     # stat[11]
        'fileno'    => undef,    # fileno()
} );
```

**Comment by atoomic** [comment]
we probably simply need to replace `s{new}{file}`

**Comment by toddr** [comment]
This has been resolved by 406fb0cddf51d5c43158f742a97fa54d62589753
